backend/fms_core/services/process_measurement.py

```diff
@@ -1,10 +1,11 @@
 from datetime import datetime
 from django.core.exceptions import ValidationError
 
+from fms_core.services.property_value import validate_non_optional_properties
 from fms_core.models import ProcessMeasurement
 
 
-def create_process_measurement(process, source_sample, execution_date, volume_used=None, comment=None):
+def create_process_measurement(process, source_sample, execution_date, volume_used=None, comment=None, process_measurement_properties=None,):
```

Comment:
You don't use this ... It is not a bad idea but i think we don't need it. as long as the test is done inside both create_process_properties and process_measurement_properties the general result is the same. If you want to prevent unnecessary processing as soon as when creating a process_measurement it is great but you have to actually add the parameter when you call the function ... At the moment, it appear to be deadcode.